### ISSUE TITLE
Recruit instantly imprisoned

### ISSUE THREAD

**Issue by EelSoul** [OrionFive/Hospitality#37]
I just recruited a guest and instantly after she decided to stay my pawn took her to prison. She has a 0.5% recruitment chance now. I guess instead of making this gal a part of the colony the game recognized her as a prisoner and my attempt to recruit her as recruiting a prisoner or smth. Maybe the first action, which already has been succesful, has been interrupted by some other action that the pawn has taken, which lead the second recruitment attempt to be misinterpreted. Weird stuff, hope you look into that. Great mod otherwise.

**Comment by OrionFive** [comment]
Sorry, I wouldn't know where to start reproducing this issue. I don't think there's much I can do here...

**Comment by EelSoul** [comment]
What's really weird is that it was just a one time ocurrence and never happened since, so I have no actual idea what caused it.

**Comment by OrionFive** [comment]
Yeah those bugs are practically impossible to fix.

**Comment by EelSoul** [comment]
It happened again... kind of. Some people came, I succesfully recruited one of them, but he didn't become a part of the group. Instead, one of the guests got imprisoned and the next day everyone else, including the one I recruited, just left. Imprisonment of this one guy did not worsen my relationship with the faction, but when I tried to capture the one I originally recruited - relationship got worse. 
The last line of debug log is when the error happened, I guess. 
![dllhost_2017-02-13_11-19-18](https://cloud.githubusercontent.com/assets/25702633/22870889/69ef1e3e-f1de-11e6-9277-e3dc3d2b7ad8.png)
